### DISCUSSION TITLE
Improve user's welcome to the community

### DISCUSSION THREAD

**Discussion by lucca65** [cambiatus/frontend#664] [Ideas]
Today new users have no proper welcoming to the community: they lack knowledge about how to communicate with admins, no information about events or social media, no introduction. Cambiatus communities admins now relies on fully offline experiences to engage with their community. 

We had a few discussions and ideas about this in the past:

- An improved [footer element](https://www.figma.com/file/RSoh4DDEj0N1TVkQT0SxIn/Header-and-Footer?node-id=361%3A1768) on issue https://github.com/cambiatus/frontend/issues/300 (@juramos-2020 first issue with us if I'm not mistaken)
- An draft for an [welcome email on our backend](https://github.com/cambiatus/backend/blob/2f23b4ec388a01ca3e823447e33383d6dcbadde0/lib/cambiatus_web/email.ex#L10-L19)

During development of https://github.com/cambiatus/backend/pull/199 @adubas asked about our welcoming emails not being actively used, only implemented and this lead to this idea of improving the onboarding of our users

**Comment by lucca65** [comment]
@muguika @heltonlr @rafachadud I would love to hear your feedback about this idea!

**Reply by rafachadud** [reply]
I love those features mentioned above. And what do you @muguika @heltonlr @lucca65 think about a welcome modal for onboarding new users?
We can display a "carousel" modal, with maybe 3 messages for the user, introducing Cambiatus and the community.
